01_Data/src/layers/sequelize/model/DeviceModel/VariableAttribute.ts

```diff
@@ -78,6 +78,14 @@ export class VariableAttribute extends Model implements VariableAttributeType {
   })
   declare constant?: boolean;
 
+  @Column({
+    type: DataType.DATE,
+    get() {
+      return this.getDataValue('timestamp').toISOString();
+    },
+  })
+  declare timestamp: string;
```

Comment:
Nitpick: The naming here could be not clear enough. Since this extent sequelize Model there are already 2 timestamps automatically added (createdAt and updatedAt). With those 2 fields in mind, I would get confused if I were new a looking at the DB. Wondering what the generic timestamp represents.

Reply:
fixed!